### ISSUE TITLE
liteidex36.2-1.macos-qt5.12.5 open error on macos10.14.6

### ISSUE THREAD

**Issue by kyokuki** [visualfc/liteide#1094]
liteidex36.2-1.macos-qt5.12.5 
open error on macos10.14.6

---
```
Process:               LiteIDE [3308]
Path:                  /Applications/LiteIDE.app/Contents/MacOS/LiteIDE
Identifier:            com.visualfc.liteide
Version:               36.2 (X36)
Code Type:             X86-64 (Native)
Parent Process:        ??? [1]
Responsible:           LiteIDE [3308]
User ID:               501

Date/Time:             2019-12-23 16:40:01.272 +0800
OS Version:            Mac OS X 10.14.6 (18G95)
Report Version:        12
Anonymous UUID:        F1DB71AB-2F87-1441-0FC5-2D8C55C0752F


Time Awake Since Boot: 5500 seconds

System Integrity Protection: enabled

Crashed Thread:        0

Exception Type:        EXC_CRASH (SIGABRT)
Exception Codes:       0x0000000000000000, 0x0000000000000000
Exception Note:        EXC_CORPSE_NOTIFY

Termination Reason:    DYLD, [0x1] Library missing

Application Specific Information:
dyld: launch, loading dependent libraries

Dyld Error Message:
  Library not loaded: @rpath/QtPrintSupport.framework/Versions/5/QtPrintSupport
  Referenced from: /Applications/LiteIDE.app/Contents/MacOS/LiteIDE
  Reason: image not found

Binary Images:
       0x10ea48000 -        0x10ea4bff3 +com.visualfc.liteide (36.2 - X36) <498978C0-0618-34EF-8DC7-9251470D3DE3> /Applications/LiteIDE.app/Contents/MacOS/LiteIDE
       0x10ea51000 -        0x10eb3aff7 +libliteapp.1.dylib (36.0 - X36) <48645E06-92E9-3D54-8907-8D0414D2C5A3> /Applications/LiteIDE.app/Contents/MacOS/libliteapp.1.dylib
       0x1141b0000 -        0x11421a70f  dyld (655.1.1) <DFC3C4AF-6F97-3B34-B18D-7DCB23F2A83A> /usr/lib/dyld

Model: MacBookPro11,1, BootROM 156.0.0.0.0, 2 processors, Intel Core i5, 2.6 GHz, 8 GB, SMC 2.16f68
Graphics: kHW_IntelIrisItem, Intel Iris, spdisplays_builtin
Memory Module: BANK 0/DIMM0, 4 GB, DDR3, 1600 MHz, 0x80AD, 0x484D54343531533641465238412D50422020
Memory Module: BANK 1/DIMM0, 4 GB, DDR3, 1600 MHz, 0x80AD, 0x484D54343531533641465238412D50422020
AirPort: spairport_wireless_card_type_airport_extreme (0x14E4, 0x112), Broadcom BCM43xx 1.0 (7.77.61.2 AirPortDriverBrcmNIC-1305.8)
Bluetooth: Version 6.0.14d3, 3 services, 27 devices, 1 incoming serial ports
Network Service: Wi-Fi, AirPort, en0
Serial ATA Device: APPLE SSD SM0128F, 121.33 GB
USB Device: USB 3.0 Bus
USB Device: Apple Internal Keyboard / Trackpad
USB Device: BRCM20702 Hub
USB Device: Bluetooth USB Host Controller
Thunderbolt Bus: MacBook Pro, Apple Inc., 17.2

```

**Comment by liugangnhm** [comment]
same issue

**Comment by liugangnhm** [comment]
fixed in 36.3 for me!
